lib/msal-core/src/utils/UrlUtils.ts

```diff
@@ -115,10 +115,10 @@ export class UrlUtils {
      * @param tenantId The tenant id to replace
      */
     static replaceTenantPath(url: string, tenantId: string): string {
-        url = url.toLowerCase();
-        const urlObject = this.GetUrlComponents(url);
+        const lowerCaseUrl = url.toLowerCase();
+        const urlObject = this.GetUrlComponents(lowerCaseUrl);
```

Comment:
can we not just say: 
``` javascript
const urlObject = this.GetUrlComponents(url.toLowerCase());
```

Reply:
We can, I personally don't mind the verbosity. Easier to read imo.